mithril-relay/Cargo.toml

```diff
@@ -46,5 +46,5 @@ slog-bunyan = "2.5.0"
 slog-scope = "4.4.0"
 slog-term = "2.9.0"
 thiserror = "1.0.56"
-tokio = { version = "1.35.1", features = ["full"] }
+tokio = { version = "1.37.0", features = ["full"] }
```

Comment:
Should it be updated in the other crates as well? Or would it make more sense to do it in a separate PR?

Reply:
I updated for the relay as it was needed for the latest version of `libp2p`.
We should definitely update the other crates it in a separate PR (I added a task in #1614 for that).